Firestore/Example/Tests/SpecTests/CMakeLists.txt

```diff
@@ -12,26 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if(APPLE)
-  firebase_ios_objc_test(
-    firebase_firestore_objc_spec_test
-    HOST firebase_firestore_example_app
-    SOURCES
-      FSTLevelDBSpecTests.mm
-      FSTMemorySpecTests.mm
-      FSTMockDatastore.h
-      FSTMockDatastore.mm
-      FSTSpecTests.h
-      FSTSpecTests.mm
-      FSTSyncEngineTestDriver.h
-      FSTSyncEngineTestDriver.mm
-    DEPENDS
-      FirebaseFirestore
-      firebase_firestore_objc_test_util
-      firestore_local_testing
-      firestore_remote_testing
-    # Force this to run in the project-wide binary directory so that relative
-    # paths used during compilation are usable.
-    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
-  )
-endif(APPLE)
+if(NOT FIREBASE_IOS_BUILD_TESTS OR NOT APPLE)
```

Comment:
Same here... please confirm that checking `FIREBASE_IOS_BUILD_TESTS` is indeed intended.